benchmarks/viscoelastic_bending_beam/viscoelastic_bending_beam.prm

```diff
@@ -0,0 +1,181 @@
+
+# Model geometry (7.5x5 km, 0.025 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 75
+    set Y repetitions = 50
+    set X extent      = 7.5e3
+    set Y extent      = 5e3
+  end
+end
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# Boundary classifications
+subsection Model settings
+  set Fixed composition boundary indicators   = bottom, top, right  
+  set Fixed temperature boundary indicators   = bottom, top, left, right
+  set Prescribed velocity boundary indicators =  
+  set Free surface boundary indicators        = 
+  set Tangential velocity boundary indicators = bottom, top, right
+  set Zero velocity boundary indicators       = left
+  set Enable elasticity                       = true
+end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 4
+  set Names of fields  = stress_xx, stress_yy, stress_xy, beam
+end
+
+# Spatial domain of different compositional fields
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = 
+    set Function expression = 0; 0; 0; if (x<=4.5e3 && y>=2.5e3 && y<=3.0e3, 1, 0)
+  end
+end
+
+# Composition boundary conditions
+subsection Boundary composition model
+  set Model name = initial composition
+end
+
+# Temperature boundary and initial conditions
+subsection Boundary temperature model
+  set Model name = box
+  subsection Box
+    set Bottom temperature = 293
+    set Left temperature   = 293
+    set Right temperature  = 293
+    set Top temperature    = 293
+  end
+end
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 293
+  end
+end
+
+# Material model
+subsection Material model
+
+  set Model name = viscoelastic
+
+  subsection Viscoelastic
+    set Densities            =  2800,  2800,  2800,  2800,  3300
+    set Viscosities          = 1.e18, 1.e18, 1.e18, 1.e18, 1.e24
+    set Elastic shear moduli = 1.e11, 1.e11, 1.e11, 1.e11, 1.e10
+    set Fixed elastic time step = 1e3
+    set Use fixed elastic time step = false
+    set Use stress averaging = false 
+    set Viscosity averaging scheme = maximum composition
+  end
+
+end
+
+# Gravity model
+subsection Gravity model
+  set Model name = vertical
+  subsection Vertical
+    set Magnitude = 10.
+  end
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree           = 2
+  set Stokes velocity polynomial degree       = 2
+  set Temperature polynomial degree           = 1
+  set Use locally conservative discretization = false
+  set Use discontinuous temperature discretization = false
+  set Use discontinuous composition discretization = true
```

Comment:
Oh, so you are using discontinuous elements for the stresses as well. Did that cause any problems? Just curious.